x-pack/test/api_integration/apis/security_solution/cases_privileges.ts

```diff
@@ -0,0 +1,331 @@
+/*
```

Comment:
Why this test file is part of the `security_solution` integration tests and not of `cases`? Is it because we need to ensure that the `security_solution` can create a case? If that so why the need of that?